src/main/java/giis/model/Almacenero/AlmaceneroModel.java

```diff
@@ -8,122 +8,195 @@
-				+ "VALUES ('"+LocalDate.now().toString()+"' , 'En recogida' , '"+almaceneroId+"' ,'Ninguna');";
+		String sqlInsert = "INSERT INTO OrdenTrabajo (fecha_creacion,estado,almacenero_id,incidencia) " + "VALUES ('"
+				+ LocalDate.now().toString() + "' , 'En recogida' , '" + almaceneroId + "' ,'Ninguna');";
 		db.executeUpdate(sqlInsert);
-		
+
 	}
+
 	public List<OrdenTrabajoRecord> getOrdenesDeTrabajoDelAlmaceneroPorId(int almaceneroId) {
-		String sql="SELECT id, fecha_creacion, estado, incidencia, almacenero_id FROM OrdenTrabajo WHERE almacenero_id="+almaceneroId+";";
-		
-		List<Object[]> lista=db.executeQueryArray(sql,new Object[0]);
-		List<OrdenTrabajoRecord> listaPedidosNoRecogidos=new ArrayList<OrdenTrabajoRecord>();
+		String sql = "SELECT id, fecha_creacion, estado, incidencia, almacenero_id FROM OrdenTrabajo WHERE almacenero_id="
+				+ almaceneroId + ";";
+
+		List<Object[]> lista = db.executeQueryArray(sql, new Object[0]);
+		List<OrdenTrabajoRecord> listaPedidosNoRecogidos = new ArrayList<OrdenTrabajoRecord>();
 		OrdenTrabajoRecord dto;
-		for(Object[] d: lista) {
-			dto=new OrdenTrabajoRecord();
+		for (Object[] d : lista) {
+			dto = new OrdenTrabajoRecord();
 			dto.setId(d[0].toString());
 			dto.setFechaCreacion(d[1].toString());
 			String estado = d[2].toString();
 			switch (estado) {
 			case "En recogida" -> dto.setEstado(Estado.EnRecogida);
 			case "Pendiente de empaquetado" -> dto.setEstado(Estado.PendienteDeEmpaquetado);
-			case "Empaquetado" ->	dto.setEstado(Estado.Empaquetado);
+			case "Empaquetado" -> dto.setEstado(Estado.Empaquetado);
 			default -> throw new IllegalArgumentException();
 			}
-			if(d[3]==null) {
+			if (d[3] == null) {
 				dto.setIncidencias("Sin incidencias");
-			}else {
+			} else {
 				dto.setIncidencias(d[3].toString());
 			}
-			
+
 			dto.setAlmaceneroId(d[4].toString());
-			
+
 			listaPedidosNoRecogidos.add(dto);
 		}
-		
+
 		return listaPedidosNoRecogidos;
 	}
-	
-	public PaqueteDto creaEtiquetaYAlbaran(OrdenTrabajoRecord otr) {
-		 String sacarInfoEtiquetaEnvio = "SELECT ped.id AS pedido_id, c.nombre, c.apellidos, c.dni, ped.fecha " +
-                 "FROM OrdenTrabajoProducto otp " +
-                 "JOIN Producto p ON otp.producto_id = p.id " +
-                 "JOIN ProductosPedido pp ON pp.producto_id = p.id " +
-                 "JOIN Pedido ped ON pp.pedido_id = ped.id " +
-                 "JOIN Cliente c ON ped.cliente_id = c.id " +
-                 "WHERE otp.orden_trabajo_id ="+otr.getId()+";";
-		 
-		 List<Object[]> lista=db.executeQueryArray(sacarInfoEtiquetaEnvio,new Object[0]);	
-		  PaqueteDto dto = null;
-		 for(Object[] d: lista) {
-			 
-			 String pedidoId= d[0].toString();
-			 String nombreCliente = d[1].toString();
-             String apellidosCliente = d[2].toString();
-             String dniCliente = d[3].toString();
-             String fecha= d[4].toString();
+
+	public String creaEtiqueta(OrdenTrabajoRecord otr) {
```

Comment:
Aquí aparte de comentarios similares a los indicados más arriba, hay algo **muy importante**. No he visto en el modelo el concepto de caja/paquete (que aquí estáis asumiendo que es la etiqueta) ni de albarán. Esto deberán ser entidades que se guarden en la base de datos.